src/main/java/ZukeLogic.java

```diff
@@ -0,0 +1,99 @@
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Task;
+import task.Todo;
+import task.Event;
+import task.TaskList;
+import tool.MessageDecoder;
+import tool.ResponseManager;
+import exception.InputException;
+
+public class ZukeLogic {
+    private final TaskList taskList;
+
+    ZukeLogic() {
+        taskList = new TaskList();
+    }
+
+    public static ZukeLogic initZuke() {
+        ResponseManager.greet();
+        return new ZukeLogic();
+    }
+
+    public static void exitZuke() {
+        ResponseManager.sayGoodbye();
+    }
+
+    public void chattingStart() {
```

Comment:
Avoid long methods. Take corrective action when method exceeds 30 lines of code.

Reply:
Thank you for the advice! Will refactor the method to make it more readable